src/main/java/com/epam/lowcost/controller/LoginController.java

```diff
@@ -5,32 +5,55 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.support.SessionStatus;
 
 import java.util.Map;
 
 
 @Controller
 @RequestMapping(value = "/login")
+@SessionAttributes(value = "sessionUser")
 public class LoginController {
     @Autowired
     UserService userService;
 
+    @GetMapping
+    public String auth(@ModelAttribute("sessionUser") User user) {
+        if (user.isAdmin()) {
+            return "admin";
+        } else {
+            return "redirect:/tickets/myTickets";
```

Comment:
Why you return quite different things? Maybe simpler will be return other Views without redirect ? (or vice versa)

Reply:
I think you have to think more time how this implement. For instance you can return one JSP with mark admin or !admin. And then JSP generates correct view for each specific role

Reply:
Changed method to return jsp in both cases.